resources/tests.mjs

```diff
@@ -1113,6 +1113,63 @@ Suites.push({
     ],
 });
 
+Suites.push({
+    name: "TodoMVC-Dart2JS",
+    url: "resources/todomvc/dart2js-jaspr/index.html",
+    tags: ["todomvc"],
```

Comment:
they both should have the experimental tag set, and should be have `disabled: true,`

Reply:
Done.